# HG changeset patch
# User Rik <rik@octave.org>
# Date 1371937002 25200
#      Sat Jun 22 14:36:42 2013 -0700
# Node ID 81e041f5633ad3b617455ed4c3e4668f3eb953b5
# Parent  366a13bd71d588d89a7db300475b988da3f173db
doc: Add copyobj, struct2hdl, hdl2struct to manual.

* doc/interpreter/plot.txi: Add copyobj, struct2hdl, hdl2struct to manual.

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -526,17 +526,19 @@ the second a cosine wave.  Figure number
 @node Use of axis@comma{} line@comma{} and patch Functions
 @subsection Use of axis@comma{} line@comma{} and patch Functions
 
 You can create axes, line, and patch objects directly using the
 @code{axes}, @code{line}, and @code{patch} functions.  These objects
 become children of the current axes object.
 
 @DOCSTRING(axes)
+
 @DOCSTRING(line)
+
 @DOCSTRING(patch)
 
 @DOCSTRING(fill)
 
 @DOCSTRING(surface)
 
 @node Manipulation of Plot Windows
 @subsection Manipulation of Plot Windows
@@ -1163,16 +1165,41 @@ objects.
 @DOCSTRING(set)
 
 @DOCSTRING(ancestor)
 
 @DOCSTRING(allchild)
 
 @DOCSTRING(findfigs)
 
+@cindex saving graphics objects
+
+Figures can be printed or saved in many graphics formats with @code{print} and
+@code{saveas}.  Occasionally, however, it may be useful to save the original
+Octave handle graphic directly so that further modifications can be made such
+as modifying a title or legend.  
+
+This can be accomplished with the following functions by 
+
+@example
+@group
+fig_struct = hdl2struct (gcf);
+save myplot.fig -struct fig_struct;
+@dots{}
+fig_struct = load ("myplot.fig");
+struct2hdl (fig_struct);
+@end group
+@end example
+
+@DOCSTRING(hdl2struct)
+
+@DOCSTRING(struct2hdl)
+
+@DOCSTRING(copyobj)
+
 @node Graphics Object Properties
 @subsection Graphics Object Properties
 @cindex graphics object properties
 
 @menu
 * Root Figure Properties::      
 * Figure Properties::           
 * Axes Properties::             
